src/views/v2/month/calendar-body/day.php

```diff
@@ -108,7 +120,16 @@ class="tribe-events-calendar-month__day-cell tribe-events-calendar-month__day-ce
 	>
 		<h3 class="tribe-events-calendar-month__day-date tribe-common-h4">
 			<span class="tribe-common-a11y-visual-hide">
-				<?php echo esc_html( sprintf( _n( '%s event', '%s events', $day['found_events'], 'the-events-calendar' ), number_format_i18n( $day['found_events'] ) ) ); ?>,
+				<?php
+				echo esc_html(
+					sprintf(
+						_n( '%1$s %2$s', '%1$s %3$s', $day['found_events'], 'the-events-calendar' ),
```

Comment:
* :warning: Mismatched placeholders is probably an error
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: a608eea9ab64d2e464255f14f9fee083</sub>